index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Library</title>
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+</head>
+<body>
+  <main class="container">
+
+    <div class="row">
+
+      <div class="col-md-6 col-lg-12">
+        <h1 class="mt-4">
+          <p class="text-center">My Library</p>
+        </h1>
+        <form action="" class="add-books justify-content-center mt-5 mb-5 ">
+          <div class="form-group">
+            <label for="title">Title:</label>
+            <input type="text" id="title" name='title' class="form-control">
+          </div>
+
+          <div class="form-group">
+            <label for="author">Author:</label>
+            <input type="text" id="author" name='author' class="form-control">
+          </div>
+
+          <div class="form-group">
+            <label for="pages">Number of Pages:</label>
+            <input type="text" id="pages" name='pages' class="form-control">
```

Comment:
The input field should accept only positive numbers, isn't it?
```html
<input type="number" id="pages" name='pages' class="form-control" min='0'>
```
Is the better type.

Reply:
Good catch @shshamim63 and thanks for the approval too. 👍